### DISCUSSION TITLE
Radial progress error

### DISCUSSION THREAD

**Discussion by fraserws** [saadeghi/daisyui#1190] [General]
Whenever I use the radial progress component and try to build i get this error (nextjs):
`Type error: Type '{ "--value": string; "--size": string; "--thickness": string; }' is not assignable to type 'Properties<string | number, string & {}>'.`
![image](https://user-images.githubusercontent.com/58608538/192515922-823723c6-b470-4184-bc68-98d853a19fbb.png)

Any help appreciated,
Thanks.

**Comment by JiachengDang** [comment]
Same problem

**Comment by saadeghi** [comment]
Read this: https://curiouslychase.com/notes/fix-css-variable-keys-in-style-attributes-in-react-and-typescript/
> `style={{ "--value": string; "--size": string; "--thickness": string; } as React.CSSProperties}`

If you're using React with Typescript, this is how you should use CSS variables.
